### PR TITLE
EPMDJ-10179: implement status for packages field in system settings form

### PR DESCRIPTION
**Pull request by dmitrii-kliushentsev**

### CODE REVIEW

File: src/ui/components/panels/settings/agent-settings/system-settings-form/system-settings-form.tsx [hunk 1]
```diff
@@ -39,8 +38,14 @@ export const SystemSettingsForm = () => {
       <DarkFormGroup label={(
         <div tw="flex justify-between w-[400px]">
           Application Packages
-          {!unlockedPackages && (
-            <div onClick={() => setUnlockedPackages(true)} tw="flex items-center gap-x-2 font-regular cursor-pointer">
+          {!status.unlockedPackages && (
+            <div
+              onClick={() => setStatus({
+                ...status,
```
Comment (IliaSolovev) [inline src/ui/components/panels/settings/agent-settings/system-settings-form/system-settings-form.tsx:44]: Why you use spread operator here when status it is a object with one property that you update here

File: src/ui/components/panels/settings/agent-settings/system-settings-form/system-settings-form.tsx [hunk 2]
```diff
@@ -13,20 +13,19 @@
...
-  Icons, DarkFormGroup, Fields, Field,
-  dotsAndSlashesToSlash,
+  DarkFormGroup, dotsAndSlashesToSlash, Field, Fields, Icons, useFormikContext,
 } from "@drill4j/ui-kit";
 
 import "twin.macro";
 
 export const SystemSettingsForm = () => {
-  const [unlockedPackages, setUnlockedPackages] = useState(false);
+  const { status, setStatus } = useFormikContext();
```
Comment (IliaSolovev) [inline src/ui/components/panels/settings/agent-settings/system-settings-form/system-settings-form.tsx:24]: It looks like a hack. but i don`t know how do it another way. So leave it as it

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

File: src/ui/components/panels/settings/agent-settings/system-settings-form/system-settings-form.tsx (hunk 1 above)
Comment (dmitrii-kliushentsev) [reply src/ui/components/panels/settings/agent-settings/system-settings-form/system-settings-form.tsx:44]: it is necessary for the possibility of further expansion of the status

Comment (IliaSolovev) [reply src/ui/components/panels/settings/agent-settings/system-settings-form/system-settings-form.tsx:44]: YAGNI

File: src/ui/components/panels/settings/agent-settings/system-settings-form/system-settings-form.tsx (hunk 2 above)
Comment (IliaSolovev) [reply src/ui/components/panels/settings/agent-settings/system-settings-form/system-settings-form.tsx:24]: add types for status

**Review by IliaSolovev — APPROVED** [review]

**Pull request merged**